common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -0,0 +1,409 @@
+// Generated by CoffeeScript 1.6.1
+/* eslint-disable no-underscore-dangle */
+/* globals Logger, interpolate */
+
+(function() {
+    'use strict';
+
+    this.Sequence = (function() {
+        function Sequence(element) {
+            var self = this;
+
+            /* eslint-disable no-unused-vars */
+            // This makes the .apply(self, arguments) pattern Coffeescript uses possible below
+
+            this.removeBookmarkIconFromActiveNavItem = function(event) {
+                return Sequence.prototype.removeBookmarkIconFromActiveNavItem.apply(self, arguments);
+            };
+            this.addBookmarkIconToActiveNavItem = function(event) {
+                return Sequence.prototype.addBookmarkIconToActiveNavItem.apply(self, arguments);
+            };
+            this._change_sequential = function(direction, event) {
+                return Sequence.prototype._change_sequential.apply(self, arguments);
+            };
+            this.selectPrevious = function(event) {
+                return Sequence.prototype.selectPrevious.apply(self, arguments);
+            };
+            this.selectNext = function(event) {
+                return Sequence.prototype.selectNext.apply(self, arguments);
+            };
+            this.goto = function(event) {
+                return Sequence.prototype.goto.apply(self, arguments);
+            };
+            this.toggleArrows = function() {
+                return Sequence.prototype.toggleArrows.apply(self, arguments);
+            };
+            this.updateProgress = function() {
+                return Sequence.prototype.updateProgress.apply(self, arguments);
+            };
+            this.addToUpdatedProblems = function(problemId, newContentState, newState) {
+                return Sequence.prototype.addToUpdatedProblems.apply(self, arguments);
+            };
+            this.hideTabTooltip = function(event) {
+                return Sequence.prototype.hideTabTooltip.apply(self, arguments);
+            };
+            this.displayTabTooltip = function(event) {
+                return Sequence.prototype.displayTabTooltip.apply(self, arguments);
+            };
+
+            /* eslint-enable no-unused-vars */
```

Comment:
It was related to trying to make ESLint ok with the `.apply(self, arguments)` pattern the Coffee compiler likes. I've fixed instances of that pattern manually and removed the linter directives.